sklearn/ensemble/_forest.py

```diff
@@ -1321,15 +1329,21 @@ class RandomForestRegressor(ForestRegressor):
            The default value of ``n_estimators`` changed from 10 to 100
            in 0.22.
 
-    criterion : {"squared_error", "mse", "mae"}, default="squared_error"
+    criterion : {"squared_error", "mse", "mae", "poisson"}, \
+            default="squared_error"
+
         The function to measure the quality of a split. Supported criteria
         are "squared_error" for the mean squared error, which is equal to
-        variance reduction as feature selection criterion, and "mae" for the
-        mean absolute error.
+        variance reduction as feature selection criterion, "poisson" which
+        uses reduction in Poisson deviance to find splits, and "mae" for
+        the mean absolute error.
 
         .. versionadded:: 0.18
            Mean Absolute Error (MAE) criterion.
 
+        .. versionadded:: 0.24
```

Comment:
As info for a 2nd reviewer: `"poisson"` was made available with adding it to decision trees in v0.24. But it was not documented nor tested here for forests.

Reply:
Because it was not documented or tested, I do not think we should consider the feature "released". I think it is better to mark this as `1.0`.